### ISSUE TITLE
Date for each of the events is listed BEFORE the heading for each of the events

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2496]
## Source

https://airtable.com/appRFUc45sF4yWFut/tbl51inE74gEWlB0x/viwG9G4vAjsjQEh4g/rec5vbF3npGlOwKGM?blocks=hide

## Describe the bug

The date for each of the events is listed BEFORE the heading for each of the events.  A screen reader user navigating by headings would have to navigate UP/Backwards to know what date the heading for the event and subsequent information refers to.  Ideally, the heading is at the top of all information related to the event.  

## Screenshots

https://github.com/proudcity/wp-proudcity/assets/1569463/f11808ee-c119-4249-9b0a-d1cd70a42687

https://github.com/proudcity/wp-proudcity/assets/1569463/5d1da6d3-8b41-4b5f-a751-cbf034f16134

**Comment by curtismchale** [comment]
I think we'd simply move the heading to the top level of the wrapper outside the two wrappers that allow for right/left boxes. That does mean a change to our styles as the heading (technically) would then align to the left side above the date, at least if we took the easy way @lukefretwell .

**Comment by lukefretwell** [comment]
@curtismchale I think that's fine. I don't love the boxes and the truncated month is not great. Just a number of accessibility and mobility issues.

In a perfect world:

* Get rid of boxes, move date to above the time
* Put date, time, add to calendar in an unstyled `ul`

**Comment by kevindherman** [comment]
@lukefretwell getting rid of the box is smart. What if it went to the right instead of between title and time so then the heading text would come first and can still keep some of the "calendar" style of the bigger date in the list view. As a reminder, if they want something with only the title they can do the mini list in the event list widget. 

If the right align for the former date box is at all no good for accessibility or mobile, please disregard.

cc @curtismchale

**Comment by curtismchale** [comment]
![Image](https://github.com/user-attachments/assets/1e0f53d4-de80-4a68-a868-3c09eb6ecb16)

Here is a mockup

**Comment by kevindherman** [comment]
@curtismchale @lukefretwell The event title should be indented so it's in line with the widget title and the leftside of the date box. The alignment of the mini list is a good example of how it should be. See screenshots:
Event list (example of out of line)
![Screenshot 2024-08-20 at 4 34 48 PM](https://github.com/user-attachments/assets/6f1a797f-4335-4e99-a27f-46e84da22c86)


Mini list (example of how it should be):
![Screenshot 2024-08-20 at 4 35 00 PM](https://github.com/user-attachments/assets/c8b1923e-24a1-4b07-98b0-1d5b1d613c43)

**Comment by curtismchale** [comment]
Deployed

Composer: https://github.com/proudcity/wp-proudcity/releases/tag/2024.08.20.1153
Build: https://github.com/proudcity/proud-recipes/releases/tag/2024.08.20.1153
